Cargo.toml

```diff
@@ -51,6 +51,9 @@ goblin = "0.4"
 libc = "0.2"
 lset = "0.2"
 vdso = "0.1"
+log = "0.4"
+once_cell = "1.8"
```

Comment:
I would prefer the `spinning` crate. See https://github.com/wgwoods/enarx/pull/2

Reply:
Hm. not sure which of these to take, though - using pure static vars seems better, but.. if we have external/tenant-supplied shims then the set of *shims* is not static. Does that mean `BACKENDS` would also be mutable? If I built `enarx` without `backend-sgx` but handed it `shim-sgx`, could we build a keep with it? If so, that variable should probably be called `BUILTIN_BACKENDS` or something, right?

On the other hand, if `BACKENDS` defines which backends it's _possible_ for this `enarx` build to load shims for, then I'm cool with it as-is.